src/components/SuggestionsList/SuggestionsList.test.tsx

```diff
@@ -0,0 +1,166 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import {
+  default as SuggestionsList,
+  SuggestionsListProps,
+  SuggestionItemGroupProps,
+  SuggestionsListState,
+} from './SuggestionsList';
+
+describe('<SuggestionsList />', () => {
+  let rendered: ShallowWrapper<SuggestionsListProps, SuggestionsListState>;
+  let onItemSelected: jest.Mock<{}>;
+  let isLoading: boolean;
+  let groupedItems: SuggestionItemGroupProps[];
+  const target = document.createElement('div');
+
+  const getProps = () => {
+    return {
+      target,
+      isLoading,
+      groupedItems,
+      onItemSelected,
+      loadingText: 'loadingText',
+      noResultsText: 'noResultsText',
+      selectedId: 2,
+      searchText: 'searchText',
+    };
+  };
```

Comment:
This feels dirty. An impure function that relies on the outer values to be modified by the `beforeEach` blocks?

Better:

```js
// whatever values you think make sense here
const defaultProps = {
      target: null,
      isLoading: false,
      groupedItems: [],
      onItemSelected: () => {},
      loadingText: 'loadingText',
      noResultsText: 'noResultsText',
      searchText: 'searchText',
      selectedId: null,
};

const getProps = (overrides) => {
    return {
      ...defaultProps,
      ...overrides,
    };
};
```

Reply:
lol oops i always do this